CONTRIBUTING.md

```diff
@@ -35,9 +36,42 @@ times in N different commits. If there was some accidental reformatting or white
 changes during the course of your commits, please rebase them away before submitting
 the PR.
 
+### CHANGELOG using Beachball
+
+[Beachball](https://microsoft.github.io/beachball/) is a semantic version bumper that also has an automated tool to ask contributors to log changes in a simple CLI manner.
+
+The TeamsJS Client SDK `2.0-preview` branch contains a [changelog](./packages/teams-js/CHANGELOG.md) for substantial changes in the `<root>/packages/teams-js` directory. If you make any changes to the `<root>/packages/teams-js` directory (you can see the exception files and directories in [beachball.config.js](./beachball.config.js`), you must run `yarn changefile` from the monorepo root to generate change files.
+
+Beachball generates JSON change files based on a few simple answers from you:
+
+- Change type: this can be one of four types: Patch, Minor, None, and Major.
+
+  - Patch - bug fixes; no API changes.
+
+  - Minor - small feature; backwards compatible API changes.
+
+  - None - this change does not affect the published package in any way.
+
+  - Major - major feature; breaking changes.
+
+- Describe changes (type your own or choose one of the commit descriptions. Try to make it descriptive- it will help you if you need to locate the changefile later.)
+
+And that's it! As easy as hitting 'enter' twice. Beachball will automatically commit the change file you've created. All you have to do is run `yarn changefile` in the monorepo root to do the above change file generation as the last step in your branch to make sure your PR is ready for review. Our pipelines will check to see if you haven't generated a changefile and they will fail if that's the case, so please remember to generate a changefile and update the content accordingly.
```

Comment:
Maybe "Our pipelines will check to see if you generated a changefile and will fail if you forgot. If they do, please create the changefile as per the steps listed and update the content accordingly."

Reply:
Done!